api/src/test/java/bisq/api/http/service/auth/ApiPasswordManagerTest.java

```diff
@@ -0,0 +1,391 @@
+package bisq.api.http.service.auth;
+
+import bisq.api.http.exceptions.UnauthorizedException;
+
+import bisq.core.app.BisqEnvironment;
+
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.jetbrains.annotations.NotNull;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+
+import com.github.javafaker.Faker;
```

Comment:
I dont find that class in the source code at https://github.com/Javafaker. 
Do we really need that? It is one of those examples where I don't want to get new dependencies (unknown developer, small functionality)

Reply:
Ah sorry, just saw its a test class. So then I dont care so much about that dependency anymore, but still would like to know better what the class provides.

Reply:
Faker is random data generator. On the top level it has methods with categories.